releasenotes/notes/deprecated-positional-arguments-cb78dd22eb0894e0.yaml

```diff
@@ -0,0 +1,8 @@
+---
+issues:
+  - |
+    If positional arguments were passed into QSVR or QSVC and these classes were printed, an 	 exception was raised.
```

Comment:
Presumably this is to note that if you do pass positional arguments then and use of repr/str conversion on the instance will error. This is still the case right? I.e the issues: section is to note issues that are in the release - so for me the sentence could be changed from past to present tense. If the tense was more related to this is what was fixed then it should go in the fixes: section.